source/renderer/app/components/wallet/WalletRestoreDialog.js

```diff
@@ -380,14 +395,19 @@ export default class WalletRestoreDialog extends Component<Props, State> {
 
     const regularTabClasses = classnames([
       'regularTab',
-      !this.isCertificate() ? styles.activeButton : '',
+      !this.isYoroi() && !this.isCertificate() ? styles.activeButton : '',
```

Comment:
@DeeJayElly please replace `!this.isYoroi() && !this.isCertificate()` with `this.isRegular() || this.isLegacy()`

Reply:
Fixed